sympy/physics/pring.py

```diff
@@ -45,11 +48,19 @@ def wavefunction(n, x):
 
 def energy(n, m, r):
     """
-    Returns the energy of the state corresponding to quantum number n.
+    Returns the energy of the state corresponding to quantum number ``n``.
 
     E=(n**2 * (hcross)**2) / (2 * m * r**2)
-    here n is the quantum number, m is the mass of the particle
-    and r is the radius of circle.
+
+    Parameters
+    ==========
+
+    ``n`` :
+        The quantum number.
+    ``m`` :
+        The mass of the particle.
```

Comment:
The mass of the particle -> Mass of the particle

Reply:
Changed.